kubectl/subcommand.go

```diff
@@ -34,6 +35,8 @@ func InspectSubcommand(command string) (Subcommand, bool) {
 		return Get, true
 	case "describe":
 		return Describe, true
+	case "exec":
+		return Exec, false
```

Comment:
InspectSubcommand returns ok=true when "subcommand could be inspected", so we don't need these lines. Even if we delete them, `InspectSubcommandInfo` returns ok=false so `colorize` becomes false so it's no problem.